kevm-pyk/src/tests/integration/test_conformance.py

```diff
@@ -105,9 +108,7 @@ def test_rest_vm(test_file: Path) -> None:
 
 BCHAIN_TEST_DIR: Final = TEST_DIR / 'BlockchainTests/GeneralStateTests'
 ALL_BCHAIN_TESTS: Final = tuple(BCHAIN_TEST_DIR.glob('**/*.json'))
-BCHAIN_TESTS: Final = tuple(
-    test_file for test_file in ALL_BCHAIN_TESTS if test_file not in (list(SKIPPED_TESTS) + list(ALL_VM_TESTS))
-)
+BCHAIN_TESTS: Final = tuple(test_file for test_file in ALL_BCHAIN_TESTS if test_file not in list(ALL_VM_TESTS))
```

Comment:
Is `list()` still needed in `list(ALL_VM_TESTS))`?

Reply:
Good catch, thank you!